en/components/grids_templates/cell-editing.md

```diff
@@ -181,6 +181,22 @@ This code is used in the sample below which implements an [`IgxSelectComponent`]
 
 For more information on how to configure columns and their templates, you can see the documentation for [Grid Columns configuration](../grid/grid.md#angular-grid-column-configuration).
 
+### Excel Style Editing
+
+## Angular Grid cell editing and edit templates Example
+
+<code-view style="height:650px" 
+           data-demos-base-url="{environment:demosBaseUrl}" 
+           iframe-src="{environment:demosBaseUrl}/grid/grid-editing-excel-style" alt="Angular Grid Excel Style Editing Example">
+</code-view>
+
+Using Excel Style Editing allows the user to navigate trough the cells just as he would using the Excel, and ever so quickly edit them:
```

Comment:
Looks good but makes it sound like a built-in functionality rather than something that requires custom implementation. 
We should focus more on explaining that this is a custom implementation built on top of the grid and add the steps needed to implement it - what events need to be bound and what handling needs to be added in them, add some code snippets etc., Maybe something more similar to https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/keyboard-navigation#custom-keyboard-navigation

Reply:
updated